http-observatory/serverless.yml

```diff
@@ -1,4 +1,4 @@
-service: aws-python-scheduled-cron
+service: http-obs-cron
```

Comment:
Maybe make this `http-observatory-cron`, since you spelled out observatory elsewhere? Otherwise it looks great.